stage1/usr_from_kvm/keys/lkvm

```diff
@@ -0,0 +1,12 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Similar idea as in vagrant. sshd can be also enforced (but this would require rewriting config file in time of image preparation) to run only on particular interface/port (for example on primary interface with /31 netmask configured for... host<->vm connection purpose).